### PR TITLE
Add plugin API for vehicle g-forces

### PR DESCRIPTION
**Pull request by Phelicks**
I'm writing a plugin which displays live ride information and this API would be really helpful.

### CODE REVIEW

File: distribution/openrct2.d.ts [hunk 1]
```diff
@@ -1153,6 +1161,11 @@ declare global {
          */
         trackLocation: CoordsXYZD;
 
+        /**
+         * The current g-forces of this car.
+         */
+        gForces: GForces;
```
Comment (IntelOrca) [inline distribution/openrct2.d.ts:1167]: This should be `readonly` as you have not provided a setter.

**Comment by IntelOrca** [comment]
Thanks, finally can you increment the API version in Scripting.cpp and include a line in the changelog referencing the pull request number. We usually do this for each plugin API change.

**Comment by Phelicks** [comment]
done :)

**Comment by Gymnasiast** [comment]
I'm a bit apprehensive about merging this so close to the release, especially since we have had crashes after adding plug-in APIs before.

**Review by IntelOrca — APPROVED** [review]

**Comment by IntelOrca** [comment]
I think this is safe enough

**Review by duncanspumpkin — APPROVED** [review]

**Review by Gymnasiast — APPROVED** [review]

**Pull request merged**